### PR TITLE
Bug correction in case of non cascaded build settings

### PR DESCRIPTION
**Pull request by leclem**

### CODE REVIEW

**Comment by jeffmac-aix** [comment]
cv2 is missing from requirements

**Comment by leclem** [comment]
Is it ? I can see it if I go to https://github.com/Vikit-ai/sdk/blob/possibility-to-todefine-vikit-key-in-settings/requirements.txt
image

**Review by jeffmac-aix — APPROVED** [review]
Approve though we will fix the minor issue in the very next version

File: vikit/common/config.py [hunk 1]
```diff
@@ -30,6 +32,7 @@
 else:
     load_dotenv(dotenv_path=env_file)
 
+creation_uuid = str(uuid.uuid4())
```
[the remark this answers is not in the document]
Comment (leclem) [reply vikit/common/config.py:35]: Indeed, it is not, this is why I inject more uuids from the instanciated objects as config function parameters

File: vikit/common/config.py [hunk 2]
```diff
@@ -61,13 +64,13 @@ def get_nb_retries_http_calls() -> int:
...
 
-def get_prompt_mp3_file_name() -> str:
+def get_prompt_mp3_file_name(uuid = creation_uuid) -> str:
     """
     The name of the mp3 file either converted from user  or
     generated using an llm and that we use to extract subtitles from the video
     """
     prompt_mp3_file_name = os.getenv(
-        "PROMPT_MP3_FILE_NAME", "loose_faith_prompt_upload.mp3"
+        "PROMPT_MP3_FILE_NAME", "loose_faith_prompt_upload_" + uuid + ".mp3"
```
[the remark this answers is not in the document]
Comment (leclem) [reply vikit/common/config.py:73]: Yes totally

**Pull request merged**